Validation/HGCalValidation/plugins/HGCalTriggerValidator.cc

```diff
@@ -97,7 +102,7 @@ class HGCalTriggerValidator : public DQMGlobalEDAnalyzer<Histograms> {
 
 private:
   void bookHistograms(DQMStore::IBooker &, edm::Run const &, edm::EventSetup const &, Histograms &) const override;
-  void dqmAnalyze(edm::Event const &, edm::EventSetup const &, Histograms const &) const override;
+  void dqmAnalyze(edm::Event const &, const edm::EventSetup &, Histograms const &) const override;
```

Comment:
This is semantically the same. There is no reason to change since the convention `Type const &` is used throughout this file and not `const Type &`